gen_types.go

```diff
@@ -3562,6 +3562,20 @@ type Message struct {
 	ReplyMarkup *InlineKeyboardMarkup `json:"reply_markup,omitempty"`
 }
 
+// GetLink is a helper method to easily get the message link excluding group and private chat types.
```

Comment:
Check the first line of this file. :) 

No methods should be manually added in `gen_*` files, as regenerating the API will remove manual changes. `custom_helpers` is probably the right place for this.

Reply:
Also worth mentioning that this only returns for channels and supergroups, and will return an empty string otherwise

Reply:
> Check the first line of this file. :)
> 
> No methods should be manually added in `gen_*` files, as regenerating the API will remove manual changes. `custom_helpers` is probably the right place for this.

Ah Sorry, My bad.

Reply:
> Also worth mentioning that this method only returns links for channels and supergroups, and will return an empty string otherwise

Sure, I'll consider commenting it as well.